libs/SalesforceSDK/src/com/salesforce/androidsdk/push/PushService.java

```diff
@@ -143,40 +129,32 @@ protected void onHandleWork(Intent intent) {
 		}
 		final UserAccountManager userAccMgr = SalesforceSDKManager.getInstance().getUserAccountManager();
 		final List<UserAccount> accounts = userAccMgr.getAuthenticatedUsers();
-		try {
-            boolean register = SFDC_REGISTRATION_RETRY_INTENT.equals(intent.getAction());
-            boolean unregister = SFDC_UNREGISTRATION_INTENT.equals(intent.getAction());
-            if (register || unregister) {
-                if (allAccounts) {
-                    if (accounts != null) {
-                        for (final UserAccount userAcc : accounts) {
-
-                            /*
-                             * If 'register' is true, we are registering, if it's false, we must be
-                             * un-registering, because of the if gate above.
-                             */
-                            performRegistrationChange(register, userAcc);
-                        }
-                    }
-                } else {
-                    if (account == null) {
-                        account = userAccMgr.getCurrentUser();
-                    }
-
-                    /*
-                     * If 'register' is true, we are registering, if it's false, we must be
-                     * un-registering, because of the if gate above.
-                     */
-                    performRegistrationChange(register, account);
-                }
-            }
-        } finally {
+		boolean register = SFDC_REGISTRATION_RETRY_INTENT.equals(intent.getAction());
```

Comment:
No code changes here - just removed the wake lock handling in the `finally` block.